src/util.c

```diff
@@ -303,6 +303,22 @@ char *git__strtok(char **end, const char *sep)
 	return NULL;
 }
 
+char *git__strtok_(char **end, const char *sep, ...)
+{
+	char* res;
+	va_list seps;
+
+	va_start(seps, sep);
+
+	res = git__strtok(end, va_arg(seps, const char*));
```

Comment:
This can't be right.
`git__strtok_` is called with `buf = git__strtok_(&scan, "\r\n", "\n", "\0");`
`va_start` says that variable arguments starts after `sep`, so `seps` points to `"\n"`
Without any form of loop `git__strtok` is only called for `"\n"` and not for `"\r\n"` or `"\0"`

Reply:
Thanks, I think I misunderstood how `git__strtok` worked when I was writing this.
I've since removed it.